### ISSUE TITLE
Could it use one sentence to search google?

### ISSUE THREAD

**Issue by alice-cool** [rrwen/search_google#5]
Could it use one sentence to search google except a word?

**Comment by alice-cool** [comment]
![图片](https://user-images.githubusercontent.com/16870890/136722627-9d2f1bf6-2fa6-4683-902a-9f7b6dc895e7.png)
![图片](https://user-images.githubusercontent.com/16870890/136722692-80d615c1-9e25-4b09-aa93-19dea76096d5.png)

![图片](https://user-images.githubusercontent.com/16870890/136701660-4423a398-09b8-4a29-b5e9-64dc720ed379.png)
When running runtest, all tests failed

**Comment by rrwen** [comment]
@alice-cool Yes you should be able to use a whole sentence, the query is limited to the Google Search API limitations, but you may want to give it a try to see if it works

For the tests, see:

https://github.com/rrwen/search_google#tests
